src/components/asset-list/RecyclerAssetList2/ClaimablesListHeader.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { Animated, Easing, Image } from 'react-native';
+import CaretImageSource from '../../../assets/family-dropdown-arrow.png';
+import { useTheme } from '../../../theme/ThemeContext';
+import { ButtonPressAnimation } from '../../animations';
+import { Box, Inline, Text } from '@/design-system';
+import * as i18n from '@/languages';
+import useOpenClaimables from '@/hooks/useOpenClaimables';
+
+const AnimatedImgixImage = Animated.createAnimatedComponent(Image);
+
+const TokenFamilyHeaderAnimationDuration = 200;
+const TokenFamilyHeaderHeight = 48;
+
+const ClaimablesListHeader = ({ total }: { total: string }) => {
```

Comment:
for some reason reanimated seems to crash the app when used within RecyclerAssetList so just copy pasted this code from the Positions section which uses vanilla Animated